chain_bridge.go

```diff
@@ -200,6 +200,37 @@ func (l *LndRpcChainBridge) EstimateFee(ctx context.Context,
 	return l.lnd.WalletKit.EstimateFeeRate(ctx, int32(confTarget))
 }
 
+// SubscribeCustomMessages creates a subscription to custom messages received
+// from our peers.
+//
+// TODO(ffranr): This should be moved into its own interface because it's not
```

Comment:
+1 

Can be like `MessageTransport` or something like that, same with the below.